doc/Caliptra.md

```diff
@@ -374,14 +374,14 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
 2. HVM programs NIST compliant UDS into fuses using SoC-specific fuse programming flow. Note that this UDS goes through an obfuscation function within Caliptra IP.
 3. SoC drives the security state, which indicates that it's a manufacturing flow. See [Caliptra Security States](#caliptra-security-states) for encodings.
 4. SoC (using a GPIO pin or SoC ROM) drives BootFSMBrk (this is also used for debug cases). This can be driven at any time before cptra\_rst\_b is deasserted.
-5. SoC follows the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers.
+5. SoC follows the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers. SoC shall not attempt to initiate any mailbox operation or acquire mailbox lock via the APB interface until after the subsequent CSR flow is completed.
```

Comment:
May be easier to read if phrased in the affirmative vs negative. And does `subsequent` refer to step 6? If so, then I recommend something more direct, like the following: 

`After the CSR flow in the next step is complete, SoC may initiate mailbox operations and may acquire mailbox lock via the APB interface.`

Reply:
I will remove this sentence entirely - the restriction is repeated in step 12.